doc/source/reference/random/index.rst

```diff
@@ -0,0 +1,219 @@
+.. currentmodule:: numpy.random
+
+numpy.random
+============
+
+A `~RandomGenerator` can
+be initialized with a number of different Random Number Generators (RNG)s, and
+exposes many different probability distributions.
+
+
+Quick Start
+-----------
+
+By default, `RandomGenerator` uses normals provided by
+`xoroshiro128.Xoroshiro128` which will be faster than the legacy methods in
+`mtrand.RandomState`
+
+.. code-block:: python
+
+  # As replacement for numpy.random.RandomState
+  from numpy import random
+  random.standard_normal()
+
+`RandomGenerator` can be used as a direct replacement for
+`~RandomState`, although the random values are generated by
+`~xoroshiro128.Xoroshiro128`. The `RandomGenerator` holds an instance of a RNG.
+It is accessable as ``gen.brng``.
```

Comment:
Can we use a more descriptor name for the attribute that holds the base random number generator? Maybe `base_rng` rather than `brng`?

Reply:
It is supposed to be "Basic RNG", so it seems that brng is definitely not clear enough.

Reply:
@bashtage What is a good use of having 'access' to the brng ?

Reply:
Also, the sentence 
```
It is accessable as ``gen.brng``.
```
should perhaps rather refer to rg.brng

It might not be clear whether 'gen' refers to a namespace, variable, etc.